thirdparty/clang/.clang-format

```diff
@@ -0,0 +1,203 @@
+ContinuationIndentWidth: 4
+Cpp11BracedListStyle: true
+DeriveLineEnding: true
+DerivePointerAlignment: true
+DisableFormat:   false
+ExperimentalAutoDetectBinPacking: false
+FixNamespaceComments: true
+ForEachMacros:
+  - foreach
+  - Q_FOREACH
+  - BOOST_FOREACH
+IncludeBlocks:   Regroup
+IncludeCategories:
+  # system includes, e.g. <iostream> or <unistd.h> or <sys/types.h>
+  - Regex: '^<((linux|sys|net|netinet|arpa|crypt|rpc)\/)?([A-Za-z0-9_-]+\/?)(.h(pp)?)?>'
+    Priority: 0
+    SortPriority: 0
+  # local project (or private) includes, e.g. "Access.hpp"
+  - Regex: '^"[a-zA-Z0-9_-]+(\/[a-zA-Z0-9_-]+)*\.h(pp)?"'
+    Priority: 2
+    SortPriority: 2
+  - Regex:           '^<.*OpenSpaceToolKit\/Core.*\.hpp>'
+    Priority:        3
+    SortPriority:    3
+    CaseSensitive:   false
+  - Regex:           '^<.*OpenSpaceToolKit\/IO.*\.hpp>'
+    Priority:        4
+    SortPriority:    4
+    CaseSensitive:   false
+  - Regex:           '^<.*OpenSpaceToolKit\/Mathematics.*\.hpp>'
+    Priority:        5
+    SortPriority:    5
+    CaseSensitive:   false
+  - Regex:           '^<.*OpenSpaceToolKit\/Physics.*\.hpp>'
+    Priority:        6
+    SortPriority:    6
+    CaseSensitive:   false
+  - Regex:           '^<.*OpenSpaceToolKit\/Astrodynamics.*\.hpp>'
+    Priority:        7
+    SortPriority:    7
+    CaseSensitive:   false
+  # third party headers, e.g. <boost/algorithm/string.hpp> or <opencv2/imgproc.hpp>
+  - Regex: '^<([a-zA-Z0-9_-]+\/)+[a-zA-Z0-9_-]+\.h(pp)?>'
+    Priority: 1
+    SortPriority: 1
+IncludeIsMainRegex: ''
+IncludeIsMainSourceRegex: ''
+IndentAccessModifiers: false
+IndentCaseLabels: true
+IndentCaseBlocks: false
+IndentGotoLabels: true
+IndentPPDirectives: None
+IndentWidth:     4
+IndentWrappedFunctionNames: false
+InsertNewlineAtEOF: true
+JavaScriptQuotes: Leave
+JavaScriptWrapImports: true
+KeepEmptyLinesAtTheStartOfBlocks: false
+MacroBlockBegin: ''
+MacroBlockEnd:   ''
+MaxEmptyLinesToKeep: 1
+NamespaceIndentation: None
+ObjCBinPackProtocolList: Never
+ObjCBlockIndentWidth: 4
+ObjCSpaceAfterProperty: false
+ObjCSpaceBeforeProtocolList: true
+PenaltyBreakAssignment: 2
+PenaltyBreakBeforeFirstCallParameter: 1
+PenaltyBreakComment: 300
+PenaltyBreakFirstLessLess: 120
+PenaltyBreakString: 1000
+PenaltyBreakTemplateDeclaration: 10
+PenaltyExcessCharacter: 1000000
+PenaltyReturnTypeOnItsOwnLine: 200
+PointerAlignment: Left
+RawStringFormats:
+  - Language:        Cpp
+    Delimiters:
+      - cc
+      - CC
+      - cpp
+      - Cpp
+      - CPP
+      - 'c++'
+      - 'C++'
+    CanonicalDelimiter: ''
+    BasedOnStyle:    google
+  - Language:        TextProto
+    Delimiters:
+      - pb
+      - PB
+      - proto
+      - PROTO
+    EnclosingFunctions:
+      - EqualsProto
+      - EquivToProto
+      - PARSE_PARTIAL_TEXT_PROTO
+      - PARSE_TEST_PROTO
+      - PARSE_TEXT_PROTO
+      - ParseTextOrDie
+      - ParseTextProtoOrDie
+    CanonicalDelimiter: ''
+    BasedOnStyle:    google
```

Comment:
Do we need this?

Reply:
This is needed as it uses the defaults from the google style guide if not specified.

Reply:
This is needed as it uses the defaults from the google style guide if not specified.

Reply:
But is this proto stuff relevant for OSTk?